torch/_higher_order_ops/auto_functionalize.py

```diff
@@ -108,16 +108,22 @@ def can_auto_functionalize(op: torch._ops.OperatorBase) -> bool:
 @auto_functionalized.py_impl(DispatchKey.CompositeExplicitAutograd)
 def auto_functionalized_dense(
     _mutable_op: torch._ops.OpOverload,
+    _only_clone_tensors: Optional[Tuple[str, ...]] = None,
```

Comment:
nit: `_tensors_to_clone` ? I read `only_clone_tensors` and it reads like a boolean to me

Reply:
maybe `_only_clone_these_tensors` ?